great_expectations/data_context/data_context.py

```diff
@@ -1003,6 +1046,23 @@ def build_data_docs(self, site_names=None, resource_identifiers=None):
 
         return index_page_locator_infos
 
+    def clean_data_docs(self, site_name=None):
+         if not os.path.isdir(self.root_directory):
+            raise ge_exceptions.DataContextError(
+                "The data docs site and project root directory must be an existing directory to clean "
+            )
+
+        ge_dir = os.path.join(self.root_directory, site_name)
```

Comment:
I believe this should be using the relevant site builder and accessing its store.

Reply:
WIP at the time, looks updated

Reply:
Yup, resolved since.